benchmark/camelyon/workflows.py

```diff
@@ -108,6 +108,7 @@ def substrafl_fed_avg(
         # Keeping editable_mode=True to ensure nightly test benchmarks are ran against main substrafl git ref
         editable_mode=True,
         compile=True,
+        use_gpu=use_gpu,
```

Comment:
The renaming is not necessary, you can just set `use_gpu=torch_gpu`

Reply:
Yes, it was mainly for consistency through the workflow and to avoid having a third name. If you prefer keeping `torch_gpu` I can revert tho, I don't mind